backends/php-laravel/app/Infrastructure/Persistence/EloquentJwtHistoryRepository.php

```
@@ -0,0 +1,69 @@
+ * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
+ * @package App\Infrastructure\Persistence
+ */
+class EloquentJwtHistoryRepository extends AbstractEloquentRepository implements JwtHistoryRepository
+{
+    /**
+     * {@inheritdoc}
+     */
+    #[\Override]
+    public function findByClientId(JwtHistoryCondition $condition): Collection
+    {
+        $query = Model::query()
```

PHPStan (2.1.54)

[error] Call to an undefined static method App\Infrastructure\Models\JwtHistory::query().


(staticMethod.notFound)